core/frontend/src/tile/RealityTile.ts

```diff
@@ -266,14 +270,21 @@ export class RealityTile extends Tile {
     else
       Frustum.fromRange(this.range, scratchFrustum);
 
-    if (this.isFrustumCulled(scratchFrustum, args, true, this.boundingSphere))
+    let boundingSphere = this.boundingSphere;
+
+    if (this._reprojectionTransform) {
+      scratchFrustum.multiply(this._reprojectionTransform);
+      boundingSphere = boundingSphere.transformBy(this._reprojectionTransform, scratchSphere);

```

Comment:
Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

Reply:
Can you supply a test case? I'm concerned that the culling we do for readPixels will not account for the reprojection transform.

Reply:
I have refined this such that the reprojectionTransform is applied to the tile range, corners and bounding sphere.   I was only retaining unprojected values for the additive refinement, but I have reworked that to work from original boundary region.   All simpler and more robust now.

Reply:
> Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
> Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

I have simplified this such that the reprojection is applied directly to the stored ranges.

Reply:
> Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
> Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

This is resolved -- unnecessary with my changes as the boundingSphere is transformed to reflect the reprojection.